cmd/storage-interface.go

```diff
@@ -27,6 +27,7 @@ type StorageAPI interface {
 
 	// Storage operations.
 	IsOnline() bool // Returns true if disk is online.
+	IsLocal() bool
```

Comment:
Now that we Hostname() it is better to use that instead of IsLocal() @vadmeste

Reply:
> Now that we Hostname() it is better to use that instead of IsLocal() @vadmeste

Yes, fixed